### DISCUSSION TITLE
Update not working with out await

### DISCUSSION THREAD

**Discussion by Jarco-dev** [prisma/prisma#11116] [Q&A] [answered]
I've noticed that when running a prisma update querry I **need** to add await in front of it or else it won't update the value in the database, is this intended and if so why?

(not sure if this is the case with other types of querries too)

**Comment by austincrim** [comment, marked as answer]
Hey @Jarco-dev 👋 ,

You **have to** `await` Prisma Client operations in order for them to execute. We have this little note [in our docs](https://www.prisma.io/docs/concepts/components/prisma-client#3-use-prisma-client-to-send-queries-to-your-database):

> Note that queries will only run if used with await or chained with .then(). When using $transaction's this makes it possible for the client to pass all the queries on to the query engine as a transaction.

Let me know if you have more questions!

**Reply by Jarco-dev** [reply]
Didn't see that although admittedly it's also not that obvious but at least I know it wasn't my fault now so thanks.

**Reply by janpio** [reply]
Suggestions on where we should put this instead and make it more discoverable? Honestly open to suggestions.

**Reply by samrathchadha** [reply]
If the note was right in the starting of the CRUD api docs I think more people would see it

**Reply by Jarco-dev** [reply]
> If the note was right in the starting of the CRUD api docs I think more people would see it

I am not sure what exactly is the best place but I do think this is a far better place than as a slight note at the bottom of example code

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
